frontend/src/languages/german.js

```diff
@@ -14,39 +14,39 @@ const german = {
     password: "Kennwort",
     email: "Mail",
     confirm_password: "Passwort bestätigen",
-    uptime: "Betriebszeit",
+    uptime: "Laufzeit",
```

Comment:
```suggestion
    uptime: "Uptime",
```
Ich denke Laufzeit werden die meisten nicht verstehen.